Jenkinsfile.public

```diff
@@ -2,7 +2,7 @@ pipeline {
     agent {
         docker {
             label 'main'
-            image docker.build("storj-ci", "--pull https://github.com/storj/ci.git").id
+            image docker.build("storj-ci", "--pull git://github.com/storj/ci.git#main").id
```

Comment:
Doesn't it automatically use the default branch?

Reply:
No, it defaults to master, see here: https://build.dev.storj.io/job/storj-gerrit/10128/console